modules/arcgis/src/arcgis-deck-external-renderer.js

```diff
@@ -38,52 +40,51 @@ export default function loadArcGISDeckExternalRenderer(externalRenderers, Collec
 
   ArcGISDeckExternalRenderer.prototype.setup = setup;
 
-  function render(context) {
-    const gl = context.gl;
+  function render(renderParameters) {
+    const gl = renderParameters.context;
     const screenFbo = gl.getParameter(gl.FRAMEBUFFER_BINDING);
-
     // eslint-disable-next-line
     const dpr = window.devicePixelRatio;
     this.createOrResizeFramebuffer(
       gl,
-      Math.round(this.view.size[0] * dpr),
-      Math.round(this.view.size[1] * dpr)
+      Math.round(this.view.state.size[0] * dpr),
+      Math.round(this.view.state.size[1] * dpr)
     );
 
+    // The view state must be kept in-sync with the MapView of the ArcGIS API.
+    const state = renderParameters.state;
+
     this.deckgl.setProps({
-      layers: this.deckLayers.items.slice(),
       viewState: {
         latitude: this.view.center.latitude,
         longitude: this.view.center.longitude,
-        zoom: this.view.zoom,
-        bearing: this.view.camera.heading,
-        pitch: this.view.camera.tilt
+        zoom: this.view.featuresTilingScheme.scaleToLevel(state.scale),
+        bearing: -state.rotation,
```

Comment:
I don’t think these should be changed...

Reply:
Weird, I don't remember changing this. Maybe copy pasta.